code/modules/photography/camera/camera.dm

```diff
@@ -185,9 +186,18 @@
 	var/list/dead_spotted = list()
 	var/ai_user = isAI(user)
 	var/list/seen
-	var/list/viewlist = (user && user.client)? getviewsize(user.client.view) : getviewsize(world.view)
+	var/list/viewlist
+	var/viewc
+	if(ismob(user))
+		var/mob/mob_user = user
+		viewlist = getviewsize(mob_user.client.view)
+		viewc = mob_user.client.eye
```

Comment:
Use `?` here, not just because we can't guarantee the mob has a client, but also because clients always disappear.